web/src/locales/en-US/component.ts

```diff
@@ -81,4 +81,6 @@ export default {
   'component.global.copy': 'Copy',
   'component.global.copySuccess': 'Copy Successfully ',
   'component.global.copyFail': 'Copy Failed',
+  'component.global.modal.confirm': 'OK',
+  'component.global.modal.cancel': 'Cancel',
```

Comment:
we already have `component.global.cancel` and `component.global.confirm`, better to reuse them